hack/macros.py

```diff
@@ -1,6 +1,40 @@
 import os
+import semver
+import sys
+import traceback
+
+from github import Github
+
+# By default mkdocs swallows print() messages from macros
+def print_to_stdout(*a):
+    print(*a, file = sys.stdout)
+
+class GithubReleases:
+    def __init__(self):
+        self.tags_for_repo = {}
+        self.g = Github(os.getenv("GITHUB_TOKEN"))
```

Comment:
self.token or self.github_token would be more descriptive

Reply:
named this `self.client`